app/templates/community/index.hbs

```diff
@@ -20,6 +20,9 @@
     <p>
       <a href="https://dev.to/t/ember" target="_blank" rel="noopener noreferrer">DEV's Ember Community</a> &mdash; a great place to find interesting articles about best practices, application architecture and new learnings. Post your articles with the tag <code>ember</code>.
     </p>
+    <p>
+      <a href="https://emberwork.com/">EMBERWORK</a> &mdash; a dedicated platform for all Ember jobs, carefully curated and validated by our community members.
```

Comment:
This is overall good, but we need to be careful about things that sound like official endorsements. Something a bit more generic would be safer. Suggestion:

...a dedicated platform for Ember jobs, curated by dedicated community members.